modules/apps/27-interchain-accounts/types/validate_test.go

```diff
@@ -0,0 +1,58 @@
+package types_test
+
+import (
+	"fmt"
+
+	"github.com/cosmos/ibc-go/modules/apps/27-interchain-accounts/types"
+)
+
+func (suite *TypesTestSuite) TestValidateVersion() {
```

Comment:
want to add a test case for owner addresses which use the Delimiter, ie len(s) > 2?

Reply:
Done!